docs/manage/core.md

```diff
@@ -0,0 +1,338 @@
+# Core
+
+The core `asdf` command list is rather small, but can facilitate many workflows.
+
+## Installation & Setup
```

Comment:
Is this heading even needed on this page?

Reply:
Not really. Though "Uninstall" is on this page as well, so it made sense to bookend the "core" section with the Install / Uninstall. Technically neither Install or Uninstall should live here, but they don't belong anywhere else more than here and creating a page just for Install/Uninstall didn't seem right or fit anywhere. Update is here because it is a core command, but it only is if you installed with Git.

Reply:
One downside to this is that it moves this page up in the Google search results when someone searches for installation and setup instructions.